up-l1/mqtt_5.adoc

```diff
@@ -0,0 +1,188 @@
+
+For more information, please refer to https://mqtt.org/
+
+This document will discuss the uTransport implementation on MQTT 5.
+
+== Specifications
+
+=== UAttribute Mapping
+
+MQTT 5 supports custom header fields, and we leverage this to map the UAttribute values into the MQTT header.
+
+[cols="1,1,1,1"]
+|===
+| MQTT Header Property Code | Value (key:value) | Value Type | Description
+
+| UserProperty
+| ("0", "{UAttribute Version}")
+| String
+a| The version of the UAttribute protobuf
+
+* MUST represent the major version of the UAttribute protobuf. Currently should be set to `1`.
+
+| UserProperty
+| ("1", "{UAttribute.id}")
+| String
+a| Unique identifier for the message
+
+* MUST be set to the https://www.rfc-editor.org/rfc/rfc4122.html#section-3[hyphenated string representation] of the UUID.
+
+| UserProperty
+| ("2", "{UAttribute.type}")
+| String
+a| The type of message
+
+* MUST be the enum value for UAttributes.type link:../up-core-api/uprotocol/uattributes.proto[UMessageType enum].
+
+| UserProperty
+| ("3","{UAttribute.source}")
+| String
+a| The origin (address) of the message
+
+* MUST be the link:../basics/uri.adoc[string serialization of the UUri]
+
+| UserProperty
+| ("4","{UAttribute.sink}")
+| String
+a| The destination (address) of the message
+
+* MUST be the link:../basics/uri.adoc[string serialization of the UUri]
+
+| UserProperty
+| ("5","{UAttribute.priority}")
+| String
+a| The message's priority as defined in link:../basics/qos.adoc[QoS doc]
+
+* MUST be set to the enum value for UAttributes.priority link:../up-core-api/uprotocol/uattributes.proto[UPriority enum].
+
+| UserProperty
+| ("6", "{UAttribute.ttl}")
+| String
+a| The amount of time (in milliseconds) after which this message MUST NOT be delivered/processed anymore
+
+| UserProperty
+| ("7", "{UAttribute.permissionLevel}")
+| String
+a| The service consumer's permission level as defined in link:../up-l2/permissions.adoc#_code_based_access_permissions_caps[Code-Based uE Access Permissions (CAPs)]
+
+| UserProperty
+| ("8", "{UAttribute.commStatus}")
+| String
+a| A UCode indicating an error that has occurred during the delivery of either an RPC Request or Response message
+
+* MUST be set to the link:../up-core-api/uprotocol/ustatus.proto[UCode]'s integer value as a string.
+
+| UserProperty
+| ("9", "{UAttribute.reqId}")
+| String
+a| The identifier that a service consumer can use to correlate an RPC Repsonse message with its RPC Request
+
+* MUST be set to the https://www.rfc-editor.org/rfc/rfc4122.html#section-3[hyphenated string representation] of the UUID.
+
+| UserProperty
+| ("10", "{UAttribute.token}")
+| String
+a| The service consumer's access token
+
+| UserProperty
+| ("11", "{UAttribute.traceParent}")
+| String
+a| A tracing identifier to use for correlating messages across the system
+
+| UserProperty
+| ("upf", "{UPayload.format}")
```

Comment:
```suggestion
| ("12", "{UAttributes.UPayloadFormat}")
```

Reply:
Updated to UAttributes.payload_format to match rest of table.